docs/pages/guides/environment-variables.md

```diff
@@ -25,28 +27,24 @@ export default {
 };
 ```
 
-To use these `.extra` properties in your Expo app, you have to use the [`expo-constants`](../versions/latest/sdk/constants.md) module. Here you can see a simple component rendering the comments component, only when these are enabled.
+To use these `extra` properties in your app, you have to use the [`expo-constants`](/versions/latest/sdk/constants.md) module. Here you can see a simple component rendering the comments component, only when these are enabled.
 
-```js
+```tsx
 import Constants from 'expo-constants';
 
-export const Post = props => (
-  <View>
+export const Post = ({ enableComments = Constants.manifest.extra.enableComments || false }) => (
```

Comment:
Isn't `Constants.expoConfig` the recommended way to access this now?

Reply:
@jonsamp opened a PR to update those changes: https://github.com/expo/expo/pull/18956#pullrequestreview-1095477892

Reply:
Okay, so let's do it in this PR right away, because his PR doesn't contain this change yet.